### ISSUE TITLE
Message sequences time out for generic classes (v2)

### ISSUE THREAD

**Issue by videege** [pardahlman/RawRabbit#230]
In v2, the following works fine:

```
var response = await (_client.ExecuteSequence(p => p.PublishAsync(new HelloWorld()).Complete<HelloWorldResponse>())).Task;
```

However, if you add a generic wrapping the response class, the message on the queue doesn't get consumed and the sequence times out:

```
var response = await (_client.ExecuteSequence(p => p.PublishAsync(new HelloWorld()).Complete<Response<HelloWorldResponse>>())).Task;
```

If you don't use message sequences (instead, setting up a publisher and subscriber independently) it works fine, which leads me to believe there's a problem in the message sequence extension in how it sets up the subscriber for the response, although when I glanced at the logs it looked like it was subscribing to the correct queue.

**Comment by pardahlman** [comment]
Top of the morning, @videege 🌤 

You are absolutely right. The reason for this is that I took a shortcut when [setting the routing key](https://github.com/pardahlman/RawRabbit/blob/2.0/src/RawRabbit.Operations.MessageSequence/StateMachine/MessageSequence.cs#L165) in the message sequence. It should use naming convention, but it doesn't.

This will be fixed in the next release of 2.0!

**Comment by pardahlman** [comment]
Fixed in beta7
